src/index.js

```diff
@@ -0,0 +1,11 @@
+/*
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
+*/
+export default function (content) {
+  this.value = content;
+  const sanitizeUnicode = JSON.stringify(content)
+    .replace(/\u2028/g, '\\u2028')
+    .replace(/\u2029/g, '\\u2029');
+  return `export default ${sanitizeUnicode}`;
```

Comment:
This should definitely happen, but breaks for Angular folks (`css-loader` again 😞 )

**webpack.config.js**
```js
{
   test: /\.css$/,
   use: [ 'raw-loader', 'css-loader' /* <= Needs module.exports = '' to work */ ]
}
```

The alternative is to use `postcss-loader` standalone and ditch `raw-loader`, but also not a 💯  replacement either

Reply:
Yeah, I spent most of my eventing trying to find a way around this in the CLI without much success.

For the time being, I think we are just going to have to leave this alone & tackle the es6 export after the fact.